thruster_control/include/thruster_control/CO_VehicleSBC.h

```diff
@@ -47,25 +47,25 @@ const indextable * CO_VehicleSBC_scanIndexOD (CO_Data *d, UNS16 wIndex, UNS32 *
-extern INTEGER32 MotorVelocityFeedback;		/* Mapped at index 0x200E, subindex 0x00*/
-extern INTEGER32 MotorCurrentFeedback;		/* Mapped at index 0x2010, subindex 0x00*/
-extern UNS16 BatteryVoltage;		/* Mapped at index 0x2013, subindex 0x00*/
-extern UNS8 MotorTemperature;		/* Mapped at index 0x2019, subindex 0x00*/
-extern UNS16 SoftwareMajorVersion;		/* Mapped at index 0x2020, subindex 0x00*/
-extern UNS16 SoftwareMinorVersion;		/* Mapped at index 0x2021, subindex 0x00*/
-extern UNS16 SoftwareMicroVersion;		/* Mapped at index 0x2022, subindex 0x00*/
-extern UNS16 SoftwareBuildVersion;		/* Mapped at index 0x2023, subindex 0x00*/
-extern UNS8 BootloaderMajorVersion;		/* Mapped at index 0x2024, subindex 0x00*/
-extern UNS8 BootloaderCommonVersion;		/* Mapped at index 0x2025, subindex 0x00*/
-extern UNS8 BootloaderMinorVersion;		/* Mapped at index 0x2026, subindex 0x00*/
-extern UNS8 BootloaderBuildVersion;		/* Mapped at index 0x2027, subindex 0x00*/
-extern UNS8 BITStatus;		/* Mapped at index 0x2044, subindex 0x00*/
-extern UNS8 platformStateStatus;		/* Mapped at index 0x2051, subindex 0x00*/
-extern UNS16 MotorControllerHeartbeat;		/* Mapped at index 0x2060, subindex 0x00*/
+extern UNS8 platformState;              /* Mapped at index 0x2000, subindex 0x00*/
+extern UNS16 SBCHeartbeat;              /* Mapped at index 0x2001, subindex 0x00*/
+extern UNS8 MotorControlState;          /* Mapped at index 0x2002, subindex 0x00*/
+extern UNS8 MotorEnableDisable;         /* Mapped at index 0x2006, subindex 0x00*/
+extern INTEGER32 MotorVelocityCommand;  /* Mapped at index 0x2008, subindex 0x00*/
+extern INTEGER32 MotorVelocityFeedback; /* Mapped at index 0x200E, subindex 0x00*/
+extern INTEGER32 MotorCurrentFeedback;  /* Mapped at index 0x2010, subindex 0x00*/
+extern UNS16 BatteryVoltage;            /* Mapped at index 0x2013, subindex 0x00*/
+extern UNS8 MotorTemperature;           /* Mapped at index 0x2019, subindex 0x00*/
+extern UNS16 SoftwareMajorVersion;      /* Mapped at index 0x2020, subindex 0x00*/
+extern UNS16 SoftwareMinorVersion;      /* Mapped at index 0x2021, subindex 0x00*/
+extern UNS16 SoftwareMicroVersion;      /* Mapped at index 0x2022, subindex 0x00*/
+extern UNS16 SoftwareBuildVersion;      /* Mapped at index 0x2023, subindex 0x00*/
+extern UNS8 BootloaderMajorVersion;     /* Mapped at index 0x2024, subindex 0x00*/
+extern UNS8 BootloaderCommonVersion;    /* Mapped at index 0x2025, subindex 0x00*/
+extern UNS8 BootloaderMinorVersion;     /* Mapped at index 0x2026, subindex 0x00*/
+extern UNS8 BootloaderBuildVersion;     /* Mapped at index 0x2027, subindex 0x00*/
+extern UNS8 BITStatus;                  /* Mapped at index 0x2044, subindex 0x00*/
+extern UNS8 platformStateStatus;        /* Mapped at index 0x2051, subindex 0x00*/
+extern UNS16 MotorControllerHeartbeat;  /* Mapped at index 0x2060, subindex 0x00*/
 
-#endif // CO_VEHICLESBC_H
+#endif // THRUSTER_CONTROL_CO_VEHICLESBC_H

```

Comment:
@patapufate this is a generated file. I'd rather not touch it. It'll make diffs on update larger than necessary.

Reply:
Ok. Fixed